### PR TITLE
Nerfs oversized quirk

### PR DESCRIPTION
**Pull request by linnpap**
## About The Pull Request
Nerfs the oversized quirk, making it more of a neutral quirk to have **for the time being**. This is meant to be a temporary measure, and I've left most of the oversized quirk code commented out, as opposed to fully removed.

This removes the grab resist bonus and damage bonus.

This also adds a temporary knockdown when buckled to a chair. This is because a combat strategy I've seen is to buckle a player to a chair in order to prevent them from moving.

## How This Contributes To The Skyrat Roleplay Experience
The oversized quirk has proven itself to be controversial as all hell, mainly due to its (no offense) half-baked nature. I think that, for now, it should be nerfed to have less powergamey traits, and to be more a neutral trait. I am assuming that this will be temporary, and that eventually the trait will be sorted out balance-wise.

## Changelog

:cl:
balance: Nerfed the oversized quirk
/:cl:

### CODE REVIEW

**Comment by linnpap** [comment]
please dont kill me for this

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -177,12 +177,6 @@
 			power_throw--
 		if(HAS_TRAIT(thrown_thing, TRAIT_DWARF))
 			power_throw++
-		//SKYRAT EDIT ADDITION
```
Comment (Gandalf2k15) [inline code/modules/mob/living/carbon/carbon.dm]: No, this was to mirror the dwarfisim trait. So leave this alone.

File: modular_skyrat/modules/oversized/code/door.dm [hunk 2]
```diff
@@ -11,10 +11,10 @@
 		var/mob/living/carbon/human/crossing_human = crossed_atom
 		if(crossing_human.m_intent != MOVE_INTENT_WALK && crossing_human.body_position == STANDING_UP)
 			//We gonna bamf you, you tall fucker
-			var/affecting = crossing_human.get_bodypart(BODY_ZONE_HEAD)
-			crossing_human.apply_damage(15, BRUTE, affecting)
+			//var/affecting = crossing_human.get_bodypart(BODY_ZONE_HEAD)
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/oversized/code/door.dm]: Undo this

File: modular_skyrat/modules/oversized/code/door.dm [hunk 3]
```diff
@@ -11,10 +11,10 @@
...
+			//crossing_human.apply_damage(15, BRUTE, affecting)
 			crossing_human.Knockdown(20)
-			crossing_human.adjustOrganLoss(ORGAN_SLOT_BRAIN, 15) //We do a bit of brain damage
+			//crossing_human.adjustOrganLoss(ORGAN_SLOT_BRAIN, 15) //We do a bit of brain damage
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/oversized/code/door.dm]: Undo this too

File: modular_skyrat/modules/oversized/code/oversized_quirk.dm [hunk 4]
```diff
@@ -15,21 +15,21 @@
...
 	human_holder.dna.update_body_size()
 	human_holder.mob_size = MOB_SIZE_LARGE
-	human_holder.dna.species.punchdamagelow += OVERSIZED_HARM_DAMAGE_BONUS
-	human_holder.dna.species.punchdamagehigh += OVERSIZED_HARM_DAMAGE_BONUS
+	//human_holder.dna.species.punchdamagelow += OVERSIZED_HARM_DAMAGE_BONUS
+	//human_holder.dna.species.punchdamagehigh += OVERSIZED_HARM_DAMAGE_BONUS
 	human_holder.blood_volume_normal = BLOOD_VOLUME_OVERSIZED
-	var/speedmod = human_holder.dna.species.speedmod + OVERSIZED_SPEED_SLOWDOWN
-	human_holder.add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/species, multiplicative_slowdown=speedmod)
+	//var/speedmod = human_holder.dna.species.speedmod + OVERSIZED_SPEED_SLOWDOWN
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/oversized/code/oversized_quirk.dm]: Undo this as well

File: modular_skyrat/modules/oversized/code/oversized_quirk.dm [hunk 5]
```diff
@@ -15,21 +15,21 @@
...
 	human_holder.dna.update_body_size()
 	human_holder.mob_size = MOB_SIZE_HUMAN
-	human_holder.dna.species.punchdamagelow -= OVERSIZED_HARM_DAMAGE_BONUS
-	human_holder.dna.species.punchdamagehigh -= OVERSIZED_HARM_DAMAGE_BONUS
+	//human_holder.dna.species.punchdamagelow -= OVERSIZED_HARM_DAMAGE_BONUS
+	//human_holder.dna.species.punchdamagehigh -= OVERSIZED_HARM_DAMAGE_BONUS
 	human_holder.blood_volume_normal = BLOOD_VOLUME_NORMAL
-	var/speedmod = human_holder.dna.species.speedmod
-	human_holder.add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/species, multiplicative_slowdown=speedmod)
+	//var/speedmod = human_holder.dna.species.speedmod
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/oversized/code/oversized_quirk.dm]: Ditto

**Comment by Gandalf2k15** [comment]
Fix the above otherwise this likely won't go through, and consult others in the discord on what needs to change.

**Comment by linnpap** [comment]
im questioning whether or not to remove the ability to not be carried

**Comment by Sigmar-alkahest** [comment]
> im questioning whether or not to remove the ability to not be carried

cue 80% size kobold carrying a 12-13 foot tall brick shithouse of a being like it's nothing. i don't think that's the best idea?

File: code/game/objects/structures/beds_chairs/chair.dm [hunk 6]
```diff
@@ -133,6 +133,7 @@
 	if(HAS_TRAIT(M, TRAIT_OVERSIZED))
 		visible_message(span_warning("[src] buckles under the weight of [M] causing it to break!"))
 		playsound(src, 'modular_skyrat/modules/oversized/sound/chair_break.ogg', 70, TRUE)
+		M.Knockdown(30,TRUE)
 		deconstruct()
```
Comment (GoldenAlpharex) [inline code/game/objects/structures/beds_chairs/chair.dm]: I dunno how I feel about this, I'd rather not have the chairs being broken, to be honest it's got a lot of side-effects that weren't foreseen, like the interlink tram.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Mhmm.

**Comment by GoldenAlpharex** [comment]
They also need to be able to be carried by Bluespace Bodybags, which they currently cannot, afaik.

**Comment by Gandalf2k15** [comment]
> They also need to be able to be carried by Bluespace Bodybags, which they currently cannot, afaik.
They can

**Comment by ArdentSpark** [comment]
> > im questioning whether or not to remove the ability to not be carried
> 
> cue 80% size kobold carrying a 12-13 foot tall brick shithouse of a being like it's nothing. i don't think that's the best idea?

Wierdly people saw the difficulty of movement to be one of the most balance adjusting parts of the quirk. I.E Antagonist trying to kidnap someone, or medical trying to relocate a patient, or security trying to arrest and relocate a prisoner.

I don't believe inability to be firecarried is that bad, however it was one of the points raised most commonly. Removing the ability to grab entirely would also be a point of contention

**Comment by Gandalf2k15** [comment]
I think given we've moved this to veteran crew, these are no longer required. We will discuss it further.

**Comment by Trubbl3** [comment]
i don't like a mechanical solution to an RP issue, you can RP as someone big if you want, no need of a quirk for it, 150% should be enough

**Comment by FridayRemada** [comment]
Honestly, if it were my decision, I'd take NO upsides, but you dont bang your head on the doors. I'd sacrifice the grab buffs and everything just to have a cosmetic, slightly and OVERALL inconvenient quirk, with the upside that you get to be big.

**Comment by Gandalf2k15** [comment]
Given the fact this is now VC only, we don't need to nerf it anymore.

**Comment by Iamgoofball** [comment]
actually we do, VC only doesn't mean it's immune to being balanced, but OK

**Pull request closed without merging**